### PR TITLE
0479 cc fp complete audit

### LINKED ISSUE #479: Handle Hedera-fpcomplete-audit Issues
**Summary**
Resolve the following fpcomplete audit PRs:

- [x] https://github.com/swirlds/hedera-fpcomplete-audit/pull/153

- [x] https://github.com/swirlds/hedera-fpcomplete-audit/pull/160

- [x] https://github.com/swirlds/hedera-fpcomplete-audit/pull/159

**Possible resolution**

- Try to not hardcode constants
- compare ENUMs with`==` or `.equals()` instead of using. `.name()`

### PR DESCRIPTION
**Pull request by anighanta**
**Related issue(s)**:
Closes #479 

**Summary of the change**:
Do not use name() when comparing ENUMs with ENUMs/Strings.
Do not use hardcoded constants
ENUM variants should not be represented as strings instead of enums

**External impacts**:
None.

**Applicable documentation**
- [ ] Javadoc
- [ ] HAPI protobuf comments
- [ ] README

### CODE REVIEW

File: hapi-proto/src/main/java/com/hederahashgraph/fee/FileFeeBuilder.java [hunk 1]
```diff
@@ -279,22 +279,22 @@ public FeeData getFileContentQueryFeeMatrices(int contentSize, ResponseType resp
 
     /*
      * FileGetContentsQuery QueryHeader Transaction - CryptoTransfer - (will be taken care in
-     * Transaction processing) ResponseType - INT_SIZE FileID - 3 * LONG_SIZE
+     * Transaction processing) ResponseType - INT_SIZE FileID - BASIC_ENTITYID_SIZE
```
Comment (Neeharika-Sompalli) [inline hapi-proto/src/main/java/com/hederahashgraph/fee/FileFeeBuilder.java:282]: Can BASIC_ENTITYID_SIZE also be named as BASIC_ENTITY_ID_SIZE? If not can this be renamed to something else as BASIC_ENTITYID_SIZE and BASIC_ENTITY_ID_SIZE look alike

Comment (anighanta) [reply hapi-proto/src/main/java/com/hederahashgraph/fee/FileFeeBuilder.java]: Forgot to update the comment. fixing the typo

**Review by Neeharika-Sompalli — COMMENTED** [review]
Other files have BASIC_ENTITYID_SIZE . Those should be changed too

**Review by ljianghedera — APPROVED** [review]
LGTM.

**Review by Neeharika-Sompalli — APPROVED** [review]

**Pull request merged**